agent/src/test/resources/org/openjdk/jmc/agent/converters/test/jfrprobes_template.xml

```diff
@@ -344,6 +344,190 @@
 			</method>
 			<location>WRAP</location>
 		</event>
+		<event id="demo.jfr.convertertest.ConverterEventMultiDefaultPrimBoolean">
```

Comment:
Might as well spell out Primitive. It's long enough of a name as it is anyways. ;)